app/models/organisation.rb

```diff
@@ -25,6 +25,11 @@ def affiliated_stock_owners
       ids << id
     end
 
+    absorbed_organisations.each do |organisation|
+      ids.concat(organisation.stock_owners.pluck(:id))
```

Comment:
aren't these stock owner relationships applied to the absorbing organisation during a merge?

Reply:
Hmm, I guess if they're removed on absorbed orgs but they might still be relevant for merged 🤔

Reply:
I'm not sure what you mean - I'm saying that this line is already covered by line 33 if that helps